### ISSUE TITLE
No more d/l button under YouTube videos

### ISSUE THREAD

**Issue by yawgie** [classilla/tenfourfox#436]
Since the last TFF update the d/l button has disappeared. I tried another add-on (Easy Youtube Video Downloader Express 9.12) and was told that it had installed successfully but no d/l button either.

Any clues? Thanks!

**Comment by classilla** [comment]
Please don't open issues on Github unless you're an authorized contributor. End user support is on Tenderapp: https://tenfourfox.tenderapp.com/

To answer your question, however, this is due to the very recent YouTube site design. Any addon that relied on the old layout won't work, and updating those addons is out of scope for this project.
